app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/view_lloyd_george_is_bsol_workflow.cy.js

```diff
@@ -275,11 +275,6 @@ describe('GP Workflow: View Lloyd George record', () => {
                 ).should('be.visible');
 
                 cy.getByTestId('lg-return-btn').click();
-
-                // assert user is returned to view Lloyd George page
-                cy.contains('Lloyd George record').should('be.visible');
-                cy.getByTestId('no-records-title').should('exist');
-                cy.getByTestId('pdf-card').should('be.visible');
```

Comment:
Do we not want this in?

Reply:
locally theres no way to 'delete' a mocked PDF, so not sure why it's there in the first place